interact.py

```diff
@@ -67,6 +67,8 @@ def sample_sequence(personality, history, tokenizer, model, args, current_output
 
         logits = model(input_ids, token_type_ids=token_type_ids)
 
+        if "gpt2" == args.model:
```

Comment:
@KasparPeterson @thomwolf I think this change is no longer applicable with transformers >= 2.3.0

Reply:
I tested a model I trained with `gpt2` and then tried to interact with it, and I got this error:

```
    logits = logits[0, -1, :] / opts.temperature
IndexError: too many indices for tensor of dimension 2
```